### PR TITLE
Added blinking cursor, changed delays between texts and typing speed.

### PR DESCRIPTION
**Pull request by cringoleg**

### CODE REVIEW

**Comment by gabolaev** [comment]
cursor is not visible in light theme

**Comment by gabolaev** [comment]
let's place `createCursor` and `removeCursor` closer

**Comment by gabolaev** [comment]
default typing speed must be 86, it's important :)

File: js/app.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
-const TYPING_SPEED = 86
+const TYPING_SPEED = 70
```
Comment (gabolaev) [inline js/app.js]: no, it's must be 86. It's important :)

**Review by gabolaev — CHANGES_REQUESTED** [review]

File: js/app.js [hunk 2]
```diff
@@ -25,18 +34,20 @@ function renderText(element, resolve, text, delay) {
...
     }
 }
 
-function prepareStageElement(parentClassName, className, elementType) {
-    var element = document.createElement(elementType)
+function prepareStageElement(parentClassName, className, elementType, withCursor = true) {
+    const element = document.createElement(elementType);
     element.className = className;
-
+    if (withCursor) {
```
Comment (gabolaev) [inline js/app.js:57]:
```suggestion

    if (withCursor) {
```

File: js/app.js [hunk 3]
```diff
@@ -25,18 +34,20 @@ function renderText(element, resolve, text, delay) {
...
+        createCursor(element);
+    }
```
Comment (gabolaev) [inline js/app.js:59]:
```suggestion
    }
    
```

File: js/app.js [hunk 4]
```diff
@@ -46,26 +57,34 @@ function prepareStageElement(parentClassName, className, elementType) {
...
+    cursor.id = "cursor";
+    cursor.className = "cursor";
+    cursor.style.fontSize = refElement.style.fontSize;
+    cursor.innerHTML = "|";
+}
+
+function custom(parentClassName, className, elementType, f, withCursor = true) {
     return () => {
-        return new Promise(f(prepareStageElement(parentClassName, className, elementType)))
+        return new Promise(f(prepareStageElement(parentClassName, className, elementType, withCursor)))
```
Comment (gabolaev) [inline js/app.js]:
```suggestion
        let element = prepareStageElement(parentClassName, className, elementType)
        return new Promise(f(element))
```

looks better that way

**Review by gabolaev — CHANGES_REQUESTED** [review]

File: js/app.js [hunk 5]
```diff
@@ -85,19 +104,25 @@ run(
...
         (element) => {
             return (resolve, _) => {
                 element.href = "mailto:george@gabolaev.com";
-                renderText(element, resolve, "email me")
+                renderText(element, resolve, "email me", 2000)
             }
         },
     ),
-    text("body", "btw", "div", "btw, my name is george"),
+    text("body", "btw", "div", "btw, my name is george", 10000),
```
Comment (gabolaev) [inline js/app.js:123]: decrease delay here to 3000

Comment (gabolaev) [inline js/app.js]:
```suggestion
    text("body", "btw", "div", "btw, my name is george", 3000),
```

File: js/app.js [hunk 6]
```diff
@@ -85,19 +104,25 @@ run(
...
+    text("body", "btw", "div", "so, what are you waiting for?", 10000),
```
Comment (gabolaev) [inline js/app.js]: remove all this pReKoLs please

File: js/app.js (hunk 2 above)
Comment (silverCase) [reply js/app.js:56]: Bruh.

File: js/app.js [hunk 7]
```diff
@@ -46,26 +66,26 @@ function prepareStageElement(parentClassName, className, elementType) {
...
 }
 
 run(
-    text("body", "hello", "div", "hello"),
-    text("body", "engineer", "div", "i am a software engineer at ", 1000),
-    text("engineer", "tinkoff", "span", "Tinkoff Investments", 10),
-    text("body", "ninja", "div", "also i am 忍者 dev at "),
-    text("ninja", "kuji", "span", "KUJI Podcast", 100),
-    text("body", "about-me", "div", "i left my digital footprints here"),
+    text("body", "hello", "div", "hello,", 2000),
```
Comment (gabolaev) [inline js/app.js]:
```suggestion
    text("body", "hello", "div", "hello", 2000),
```

File: js/app.js (hunk 4 above)
Comment (cringoleg) [reply js/app.js]: This will simply not work with your architecture of method calls and promises.

Comment (cringoleg) [reply js/app.js]: I get my words back, it works, but the `withCursor` argument should remain.

**Review by gabolaev — APPROVED** [review]

**Pull request merged**